src/Kestrel.Transport.Libuv/Internal/LibuvConnectionContext.Features.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Net;
+using System.Text;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Protocols.Features;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv.Internal
+{
+    // TODO: Autogenerate like we do with frame
+    public partial class LibuvConnectionContext : IFeatureCollection,
+                                                  IHttpConnectionFeature,
```

Comment:
Inherit from something that already implements it? (e.g. have a default implementation parent class)